Dockerfile-verify-starexec

```diff
@@ -28,15 +27,15 @@ ENV FLEX_VERSION=2.6.4
 RUN wget --no-verbose "https://github.com/westes/flex/files/981163/flex-${FLEX_VERSION}.tar.gz" \
     && tar -xvzf flex-${FLEX_VERSION}.tar.gz \
     && cd flex-${FLEX_VERSION} \
-    && echo "./configure --prefix=/usr/local; make -j4; make install" | scl enable devtoolset-8 bash \
+    && ./configure --prefix=/usr/local; make -j4; make install \
     && cd .. \
     && rm -rf flex-${FLEX_VERSION} \
     && rm flex-${FLEX_VERSION}.tar.gz
 
-RUN echo "git clone --single-branch https://github.com/SMT-COMP/scrambler.git \
-        && cd scrambler \
-        && make" | scl enable devtoolset-8 bash \
-    && install ./scrambler/scrambler /usr/local/bin \
+RUN git clone --single-branch https://github.com/SMT-COMP/scrambler.git \
+    && cd scrambler \
+    && make
+RUN install ./scrambler/scrambler /usr/local/bin \
```

Comment:
Do you need this extra RUN step? Can't you merge it with the previous one, as was done before?

Reply:
No, because back then, `cd` was run in a subshell (using the `echo "..." | scl ...` technique), now it is not so `./scrambler/scrambler` would not be valid. It would be necessary to add `cd ..`

Reply:
Adding `cd ..` is a better solution IMO. Each RUN command creates a docker layer and I don't see any reason to split this into two.